src/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -113,15 +113,15 @@ private void decodeParameter(Position position, int id, ChannelBuffer buf, int l
             case 19:
                 position.set("axisZ", readValue(buf, length, true));
                 break;
+            case 21:
+                position.set("gsmLevel", readValue(buf, length, false));
```

Comment:
We usually use `RSSI` key. Is it something different?

Reply:
I noticed that in other protocols RSSI usually value from 0 to 31 even in this GH3000, but here it is value in range 1 - 5.
We can multiply it on `6,2` to fit the range 0-31

Reply:
I don't think we need to match ranges. RSSI is not really standardised for other protocols.